node/go/dev-mode/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I don't think we need to create and publish any package for that.

Can't we simply have test files (with short doc) in `node/test/go/dev-mode` ?

Reply:
So I was planning on publishing the dev-mode extension using the same tag -> publish action we are using on the other layers. (See `dev-mode-layer-publish.yml` file) so I was thinking I should just use this package.json file to trigger that.

Is there some other way I should be creating the tags for the dev mode extension to publish the layer?

Reply:
But it's a Go package. I take that if it's to be published anywhere it's to some Go registry (not npm) and we should publish what's at `go` folder, and integration tests being at `node` should not be part of the publication (similarly as we do not publish node packages with tests)

Reply:
Okay I will remove this 👍

Reply:
Well actually I think we are at one other issue running the integration tests along with the other node tests.

We don't want to release the go extension unless integration tests pass so we need to trigger the publish via this workflow somehow and I don't think I can use the [workflow run trigger](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) since this does more than just test the go extension.

Maybe I should just create a separate workflow file for the go integration tests?

Reply:
Usual configuration for integration CI is:
- Run integration tests
- If above passes and version of package has changed publish a tag

And its tag job that publishes the release.

So the trigger for publication is the actual version change in the package configuration file. What is the `package.json` equivalent in Go packages? Do we have one configured for `dev-mode` Go package here?

Reply:
From my understanding there is not really a `package.json` version equivalent for go. But I am fairly new to go so I could be wrong. I could just create a file that we add the version to and adjust the version manually 🤷 

Maybe @Mmarzex knows of a way to set a project version for go?

Reply:
The go docs just mention using a [git tag to manage the version](https://go.dev/doc/modules/publishing)

Reply:
Just confirmed with Roberto that versions are typically managed via git for go projects 😢

Reply:
Maybe I just create a file that we manually add the version number to so that we can use that to reference whether or not we should create a release tag? 🤷

Reply:
In such case maybe we simply introduce plain `VERSION` file that will have just version content as:

```
0.1.5
```

It'll allow us to rely on the same flow as with Node packages, which is quite convenient.

Reply:
It looks we both presented same idea at the same time :)

Reply:
Looking through all the tests you created has turned me into you 😎 Next thing you know I will have beautiful flowers outside the window I work next to 😆

Reply:
Alright I added a `version.json` file to the `go/packages/dev-mode` folder. Let me know what you think 🙏